modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,94 @@
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the machine set manifest file and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets for extending regular worker nodes to edge worker nodes. 
+* You ensured that your machine set manifest file specifies `node-role.kubernetes.io/edge` for the `taint` parameter.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations for your edge worker node, so that the scheduler can schedule any pods with matching taints to the cluster.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_zone_namespace> <3>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <4>
+    spec:
+      nodeSelector: <5>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <6>
+      - key: "node-role.kubernetes.io/edge"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+----
+<1> `kind`: Defines the `deployment` resource.
+<2> `name`: Specifies the name of the service. 
+<3> `namespace:` Defines the namespace that the `service` resource targets.
+<4> `machine.openshift.io/zone-group`: Defines the group to where a zone belongs. For example, `us-east-1-nyc-1` indicates the region code, `nyc`, and identifiers for a Local Zone.
+<5> `nodeSelector`: Targets edge worker nodes that match the specified toleration.
+<6> `tolerations`: Sets the tag where the scheduler can apply a taint to affected nodes. The `key:` value must match the taint value that you specified in the machine set manifest file.
```

Comment:
IMHO you don't need to explain what the toleration/taint is, or you can specify our documentation for it[1]. See how logging is used it[2]:
```
- This toleration is added to the Elasticsearch pods.
```

[1] https://docs.openshift.com/container-platform/4.12/nodes/scheduling/nodes-scheduler-taints-tolerations.html
[2] https://docs.openshift.com/container-platform/4.12/logging/config/cluster-logging-tolerations.html

Reply:
I could add a link, but the short callout would help for the purposes of this section and the example we're trying to detail.

Reply:
@dfitzmau my proposal was not to add the link, was to show how our documentation is used when referring to tolerations, a short sentence would be preferable as our goal is not to explain what toleration is, but that we are using it.
I suggest being direct too, something like:
```
<6> `tolerations`: Sets the values that match with the `taints` defined on the MachineSet manifest for the Local Zone node.
```

Reply:
Updated.

Should I remove the following links?

 * xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc[Understanding taints and tolerations]

* xref:../logging/config/cluster-logging-tolerations.adoc[Using tolerations to control OpenShift Logging pod placement]